### PR TITLE
docs(Message): add example for message.reply

### PR DESCRIPTION
**Pull request by Rexogamer**
**Please describe the changes this PR makes and why it should be merged:**
This PR adds an example for `message.reply`.

**Status and versioning classification:**
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/Message.js [hunk 1]
```diff
@@ -574,6 +574,11 @@ class Message extends Base {
    * @param {MessageOptions|MessageAdditions} [options] The additional options to provide
    * @param {MessageResolvable} [options.replyTo=this] The message to reply to
    * @returns {Promise<Message|Message[]>}
+   * @example
+   * // Reply to a message
+   * message.reply('This is a reply!')
+   *   .then(msg => console.log(`Replied to message "${msg.content}"`))
```
Comment (Rexogamer) [inline src/structures/Message.js:580]: Is this the right info to log?

Comment (advaith1) [reply src/structures/Message.js:580]: this would log `Replied to message "This is a reply!"`, is that intentional?

Comment (Rexogamer) [reply src/structures/Message.js:580]: That was what I was intending, although I wasn't sure if it was the right thing to log

Comment (Rexogamer) [reply src/structures/Message.js]: Fixed in ad778e8e046f2abec3bb1a90ad20a695359551e0

File: src/structures/Message.js [hunk 2]
```diff
@@ -574,6 +574,11 @@ class Message extends Base {
    * @param {MessageOptions|MessageAdditions} [options] The additional options to provide
    * @param {MessageResolvable} [options.replyTo=this] The message to reply to
    * @returns {Promise<Message|Message[]>}
+   * @example
+   * // Reply to a message
+   * message.reply('This is a reply!')
+   *   .then(console.log(`Replied to message "${message.content}"`))
```
Comment (papaia) [inline src/structures/Message.js:580]:
```suggestion
   *   .then(() => console.log(`Replied to message "${message.content}"`))
```

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**